sdk/extensions/Microsoft.Extensions.Azure/src/Internal/EventSourceEvent.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+
+namespace Microsoft.Extensions.Azure
+{
+    internal readonly struct EventSourceEvent: IReadOnlyList<KeyValuePair<string, object>>
```

Comment:
Why the split and visibility change?

Reply:
this is extracted from LogForwarder to be reused as a shareable source. It has to be internal to be a top-level struct

Reply:
based on the offline discussion, moving this class to Azure.Core.Shared